config/lib/express.js

```diff
@@ -82,6 +82,17 @@ module.exports.initMiddleware = function (app) {
 
 		// Disable views cache
 		app.set('view cache', false);
+
```

Comment:
@marklise you might want to consider using the CORS middleware created by the Express team -
 https://github.com/expressjs/cors

```javascript
var cors = require('cors');
var app = express();
...
app.use(cors());
```

Reply:
The express CORS lib is likely very solid. However, since we're not using this code in Production, I think adding these few lines for the _Dev_ environment will suffice.